c/ldv-memsafety/memleaks-notpreprocessed/memleaks_test3-1.c

```diff
@@ -12,7 +12,7 @@
 
 //unsafe: free without alloc (free not allocated memory)
 int *h_unsafe() {
-	return (int *)__VERIFIER_nondet_pointer();
+	return (int *)__VERIFIER_nondet_ulong();
```

Comment:
```suggestion
	return (int *)__VERIFIER_nondet_int();
```

Reply:
Why would that be preferable? The `unsigned long`/`__VERIFIER_nondet_ulong` should make sure that all bits of the address are non-deterministic.